lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1798,6 +1797,7 @@ namespace Js
         if (!scriptContext->IsJsBuiltInEnabled())
         {
             builtinFuncs[BuiltinFunction::JavascriptArray_IndexOf] = library->AddFunctionToLibraryObject(arrayPrototype, PropertyIds::indexOf, &JavascriptArray::EntryInfo::IndexOf, 1);
+            /* No inlining                Array_Sort               */ library->AddFunctionToLibraryObject(arrayPrototype, PropertyIds::sort,            &JavascriptArray::EntryInfo::Sort,              1);
```

Comment:
I have not added this to the builtinFuncs object - I'm unsure if that is necessary - seems to be just for referencing methods from intl.js?

Reply:
@jackhorton ?

Reply:
builtinFuncs is, I think, how the JIT gets information for CallDirecting native functions -- "inlining", technically, but not in the same way as inlining a JS function.